config/cmake/scripts/HDF5config.cmake

```diff
@@ -163,7 +163,7 @@ if (NOT DEFINED HPC)
       set (SITE_COMPILER_NAME "vs2012")
       set (SITE_COMPILER_VERSION "11")
     else ()
-      message (FATAL_ERROR "Invalid BUILD_GENERATOR must be - Unix, VS2017, or VS201764, VS2015, VS201564, VS2013, VS201364")
+      message (FATAL_ERROR "Invalid BUILD_GENERATOR must be - Unix VS2019, VS201964, VS2017, or VS201764, VS2015, VS201564,")
```

Comment:
comma should be removed at end, 'or' moved to before  'VS201664', and comma missing after 'Unix'?
```suggestion
      message (FATAL_ERROR "Invalid BUILD_GENERATOR must be - Unix, VS2019, VS201964, VS2017, VS201764, VS2015, or VS201564")
```

Reply:
Fixed